### PR TITLE
Ogc 674 subpage images

### PR DESCRIPTION
**Pull request by BreathingFlesh**
Please fill in the commit message below and work through the checklist. You can delete parts that are not needed, e.g. the optional description, the link to a ticket or irrelevant options of the checklist.

## Commit message

Town6: Add option for images on subpages

TYPE: Feature
LINK: OGC-674

## Checklist

- [x] I made changes/features for both org and town6
- [x] I have performed a self-review of my code
- [x] I considered adding a reviewer
- [x] I have tested my code thoroughly by hand
    - [x] I have tested styling changes/features on different browsers
- [x] I have updated the PO files

### CODE REVIEW

**Comment by BreathingFlesh** [comment]
The image field is now visible on all page-forms. For example on feriennet pages. It just doesn't do anything there. Can and should I hide it somehow?

File: src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po [hunk 1]
```diff
@@ -376,12 +376,18 @@ msgstr "La première ligne de l'adresse"
...
+msgstr "Type de marqueur"
 
 msgid "Marker"
 msgstr "Marqueur"
 
+msgid "Icon"
+msgstr "Icône"
+
+msgid "Numbers"
+msgstr ""
```
Comment (msom) [inline src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po:389]: Translation is missing

**Review by msom — CHANGES_REQUESTED** [review]
Looks good except some missing translations

**Comment by msom** [comment]
> The image field is now visible on all page-forms. For example on feriennet pages. It just doesn't do anything there. Can and should I hide it somehow?
Yes, would be probably better.

**Pull request merged**